explainer.md

```diff
@@ -17,86 +15,140 @@ The two drafts apply to both TLS server certificates, where the server authentic
 
 ## Introduction
 
-[TLS](https://www.rfc-editor.org/rfc/rfc8446) uses [X.509 certificates](https://www.rfc-editor.org/rfc/rfc5280) to associate the authenticating party's, or *subscriber's*, TLS key with its application identifiers, such as DNS names. These associations are signed by some certificate authority (CA). The peer, or *relying party*, curates a set of CAs that are trusted to only sign correct associations, which allows it to rely on the TLS to authenticate application identifiers. Typically the subscriber is the server and the relying party is the client.
+While they differ in approach, both drafts aim to solve the same problem: **enable [TLS](https://www.rfc-editor.org/rfc/rfc8446) implementations to efficiently authenticate to peers with diverse [certificate](https://www.rfc-editor.org/rfc/rfc5280) policies.** This avoids a conflict between service availability and user security.
```

Comment:
This sort of reads like a requisite lead-in paragraph was deleted, and if you squint, this still reads like the goal of a solution more than a problem statement. I think the conflict between availability and security might better be described as a corollary of the below problem statement, but I'm open to iterating over this if it's headed in the right direction.

==========
Background:
Today, TLS subscribers are provisioned with one or more certification paths which they use to authenticate themselves to TLS relying parties. Currently, TLS subscribers lack a reliable way of knowing which among these provisioned paths will be accepted by a given relying party, if any. Absent a clear signal, TLS subscribers are forced to guess, which causes connection failures when this guess is incorrect.

In the scenario in which a TLS subscriber wishes to authenticate itself to relying parties with distinct or mutually-exclusive trust settings, there is no single correct answer. Advanced TLS implementations may attempt to use TLS fingerprinting to make informed guesses; however, this is unreliable and client trust may change over time, causing existing fingerprinting mechanisms and the subsequent certification path selection to fail. 

Problem:
Trust Anchor IDs and Trust Expressions present two different approaches to solve the same problem: eliminating the uncertainty around what relying parties trust so that TLS servers can obtain, and intelligently serve, a trusted certification path to TLS relying parties.

Reply:
I generally like this, other than some degree of discomfort between "today subscribers are provisioned witth one or more" - I think we risk the reader believing today it isn't the case and most of the time it's only one.. yeah I know you say one or more, but I think a lot of our readership is stuck in a mode of "who cares what we have today works fine"

Reply:
"1 or more" is necessary for the fingerprinting --> cert selection to make sense. Perhaps we can more clearly emphasize that guesses and resulting breakage do happen, if that assuages your concerns about people thinking this doesn't actually happen today (which it does).

Reply:
I wanted to try a consistent framing between "this is make `certificate_authorities` more useful" and "this is straight-up impossible". I think the former is a better framing because a lot of deployments *can* use `certificate_authorities` today. Like it's just fine for client certs.

How's this new version?